.github/workflows/tutorials.yml

```diff
@@ -90,20 +90,28 @@ jobs:
 
     - name: Seismic notebooks
       run: |
-        $RUN_CMD py.test --nbval $SKIP examples/seismic/tutorials/
+        $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:seis_coverage.xml $SKIP examples/seismic/tutorials/
```

Comment:
out of curiosity, what are the extra options for?

Reply:
Same as in `pytest` one, basically config, report format and name